docs/source/python/models/classification.md

```diff
@@ -1,5 +1,22 @@
 # Classification
 
+## Description
+
+The `ClassificationModel` is the OpenVINO wrapper for models exported from [OpenVINO Training Extensions](https://github.com/openvinotoolkit/training_extensions). It supports multi-label classification as well as hierarchical classification.
+
+## OpenVINO Model Specifications
```

Comment:
That's not OV model specification actually, I'd say it's model wrapper i/o specs. Could you consider renaming?